webapp/Connector/src/controller/Group.js

```diff
@@ -120,33 +116,17 @@ Ext.define('Connector.controller.Group', {
     createView : function(xtype, context) {
         var v; // the view instance to be created
 
-        if (xtype === 'groupsave') {
-
-            var state = Connector.getState();
-
-            v = Ext.create('Connector.view.GroupSave', {
-                hideSelectionWarning: state.hasSelections(),
-                isMabGroup: context ? context.isMabGroup : false
-            });
-
-            state.on('selectionchange', v.onSelectionChange, v);
-
-            this.application.on('groupsaved', this.onGroupSaved, this);
-            this.application.on('groupedit', this.onGroupEdit, this);
-            this.application.on('mabgroupsaved', this.onMabGroupSaved, this);
-            this.application.on('mabgroupedited', this.onMabGroupEdited, this);
-        }
-        else if (xtype === 'groupsummary') {
+        if (xtype === 'groupsummary') {
 
             v = Ext.create('Connector.view.GroupSummary', {
-                store : StoreCache.getStore('Connector.app.store.Group'),
+                store : this.getGroupStore(),
                 groupId: context.groupId
             });
         }
         else if (xtype === 'mabgroupsummary') {
 
             v = Ext.create('Connector.view.MabGroupSummary', {
-                store: StoreCache.getStore('Connector.app.store.Group'),
+                store: this.getGroupStore(),
                 groupId: context.groupId
             });
         }
```

Comment:
I haven't tested it extensively, but adding this snippet of code fixes the navigation issue we noticed from mabsummary:

            var vm = this.getViewManager();

            vm.on('beforechangeview', function(controller, view, currentContext) {
                if (currentContext.view === "mabgroupsummary") {
                    Connector.getService('FilterStatus').activateContainer('filterstatuscontainer');
                }
            });